src/shared/components/gene-search-panel/GeneSearchPanel.module.css

```diff
@@ -66,9 +67,10 @@
   border-bottom: 1px solid var(--color-medium-light-grey);
 }
 
-/* Species column */
-.resultsTable td:first-child {
-  width: 60%;
+.resultsTable td:first-child,
+.resultsTable td:nth-child(2),
+.resultsTable td:nth-child(3),
+.resultsTable td:nth-child(4) {
```

Comment:
Is this basically saying `:not(:last-child)`? If yes, then I think `:not(:last-child)` would express the intent better.